### DISCUSSION TITLE
多账号和分享不能同时存在

### DISCUSSION THREAD

**Discussion by oldfishoh** [GaiZhenbiao/ChuanhuChatGPT#549] [General]
开启多账户后，无法分享链接
单账户下可以（config user为空）

**Comment by Keldos-Li** [comment]
好像还真是这样，分享的gradio链接，在config中有两个及两个用户时会 504 Gateway Time-out，不设用户或设置单用户时正常可用。

**Reply by oldfishoh** [reply]
能解决吗  大佬
